app/views/npq_separation/admin/finance/statements/show.html.erb

```diff
@@ -19,6 +19,10 @@
       <strong>Payment run:</strong>
       <%= boolean_red_green_tag(@statement.output_fee) %>
     </p>
+    <p class="govuk-body govuk-!-margin-bottom-2">
+      <strong>Milestones:</strong>
+      <%= @statement.milestones.pluck(:declaration_type).uniq.join(", ") %>
```

Comment:
You can use `DISTINCT` to fetch less data from the database and then humanize the types?

```ruby
@statement.milestones.distinct.pluck(:declaration_type).map(&:humanize).join(", ")
```